### ISSUE TITLE
Needs Documentation

### ISSUE THREAD

**Issue by thelink2012** [DK22Pac/plugin-sdk#4]
If there's a issue, is that we need some documentation.
[Doxygen](http://www.doxygen.nl/) is a good alternative for documenting libraries, we just need to make sure the methods, members, variables, etc, are properly documented with [Doxygen style comments](http://www.doxygen.nl/manual/docblocks.html).

Then we just need to [run Doxygen](http://www.stack.nl/~dimitri/doxygen/manual/starting.html) and done, we have a documentation.

Yeah, the hard part is documenting the already existing code... so make sure any new code has proper documentation.

**Comment by Whitetigerswt** [comment]
bumping this. The real holdback for me on using a project like this is the lack of documentation. I can look at the examples all day but that doesn't help me get started.

**Comment by Alf21** [comment]
I agree with Whitetiger. This pluginSDK is so awesome and unique. I studied it many times but it is hard to handle the functions and come along with the features, because I really don't know what are already possible. So a documentation could help to handle it. Maybe you can create a new branche or something to work together on the doc.

**Comment by Kewun0** [comment]
Awesome, unique, but what i really needed was the feature for gta3.exe 1.1 support.
Documentation would be cool too because the pluginsdk is somewhat useless without it.

**Comment by Whitetigerswt** [comment]
Another bump - I'm willing to help out in any way I can.
